templates/common/config/00-config.conf

```diff
@@ -41,9 +41,14 @@ default_backend=default_backend
 [keystone_authtoken]
 www_authenticate_uri={{ .KeystonePublicURL }}
 auth_url={{ .KeystoneInternalURL }}
-auth_type=password
+auth_type={{ if .UseApplicationCredentials }}v3applicationcredential{{ else }}password{{ end }}
```

Comment:
to keep things cleaner here I would try:
```
{{ if .UseApplicationCredentials -}}
auth_type = v3applicationcredential
application_credential_id = {{ .ACID }}
application_credential_secret = {{ .ACSecret }}
{{ else -}}
auth_type = password
username={{ .ServiceUser }}
password = {{ .ServicePassword }}
{{- end }}
```
your version is correct, but this would improve readability, other than saving an `if`